docs/package.json

```diff
@@ -29,4 +29,4 @@
     "start-server-and-test": "^2.0.0",
     "unist-util-visit": "^5.0.0"
   }
-}
+}
```

Comment:
is this an accidental change? seems git is pointing the "LF will be replaced by CRLF" type of change

Reply:
> is this an accidental change? seems git is pointing the "LF will be replaced by CRLF" type of change

besides this the file is proper 👍  as per [Adding a new language to Starlight’s docs](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md#adding-a-new-language-to-starlights-docs)

Reply:
It seems Prettier removed a line break there